README.md

```diff
@@ -51,7 +63,12 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var pizza {
+  temperature: 70,
+  bake: function() {
+    temperature = 300;
```

Comment:
Almost. You need to reference the object's temperature property. Here it looks like you're actually defining a global variable `temperature` and setting it to `300`.